vm/actor_interface/src/builtin/power/mod.rs

```diff
@@ -30,6 +30,16 @@ pub enum State {
     V6(actorv6::power::State),
 }
 
+/// Converts any `FilterEstimate`, e.g. `actorv0::util::smooth::FilterEstimate` type into
+/// generalized one `crate::FilterEstimate`.
+macro_rules! convert_filter_estimate {
+    ($from:expr) => {
+        FilterEstimate {
+            position: $from.position.clone(),
+            velocity: $from.velocity.clone(),
+        }
+    };
+}
```

Comment:
Nice.

Reply:
It was supposed to be more arcane but I gave up (for now). :(